docker-storage-setup.sh

```diff
@@ -888,6 +895,17 @@ else
   fi
 fi
 
+# Main Script
+
+if [ $# -gt 0 ]; then
+    if [ "$1" == "reset" ]; then
```

Comment:
This name looks pretty odd to me. I prefer that we continue to call it "reset". If need be, just output a message saying that one will need to cleanup docker metadata also.